### DISCUSSION TITLE
Itens especiais de eventos

### DISCUSSION THREAD

**Discussion by austv** [leogianfagna/IssueTracker#934] [Ideia]
### Servidor afetado

Todos

### Qual o propósito da ideia?

Incentivar a participação em eventos

### Descrição da ideia

Entregar itens especiais com a vitória de eventos, como Machado de Gladiador, Besta especial, etc.

### Aplicação

Para não deixar algo fácil, podemos criar um item para eventos mais especiais como Besta Treta, Batalha em Marte, Gladiador Anônimo, etc. Assim, cada evento tem o seu item especial, com skin. No caso do Vanillew, o item vem com encantamentos elevados.

Na tag do item, mostra que é um item de evento super raro. Para não ser tão fácil a obtenção, podemos criar algo de que precisa vencer aquele evento 3 vezes para conquistar.

### Referências

Servidores que entregam esses itens ao vencer eventos importantes.

**Comment by leogianfagna** [comment]
ideia: ter a tag "ausevento" em vez de "super raro"
